exporters/otlp/metrics/src/main/java/io/opentelemetry/exporter/otlp/metrics/OtlpGrpcMetricExporter.java

```diff
@@ -78,7 +80,35 @@ public void onSuccess(@Nullable ExportMetricsServiceResponse response) {
 
           @Override
           public void onFailure(Throwable t) {
-            logger.log(Level.WARNING, "Failed to export metrics", t);
+            if (t instanceof StatusRuntimeException) {
```

Comment:
Can't you use fromThrowable? I think it handles both runtime and checked versions.

https://grpc.github.io/grpc-java/javadoc/io/grpc/Status.html#fromThrowable-java.lang.Throwable-

Reply:
aha! I missed that. I'll update (although functionally it'll be the same, I think)

Reply:
done